server/mongo.js

```diff
@@ -0,0 +1,62 @@
+
+const express = require('express');
+const app = express();
+const port = 3000;
+
+const MongoClient = require('mongodb').MongoClient;
+const assert = require('assert');
+const url = 'mongodb://localhost:27017';
+const collection = 'airbnb_nearby_residences';
+var db;
+
+MongoClient.connect(url, function(err, client) {
+  assert.equal(null, err);
+  console.log("Connected successfully to server");
+  db = client.db(collection)
+
+  app.get('/:id', (req, res) => {
+    var id = Number(req.params.id);
+    findLatitudeAndLongitude(db, id, (data) => {
```

Comment:
In the spirit of separation of concerns, you may want to re-organize the code and move the DB queries into a different file, and have this handle only the GET request.